src/main/java/org/folio/rest/impl/ModTenantAPI.java

```diff
@@ -17,10 +19,13 @@
 import java.util.Map;
 
 public class ModTenantAPI extends TenantAPI {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ModTenantAPI.class);
 
   private static final long DELAY_TIME_BETWEEN_CLEANUP_VALUE_MILLIS = 3600_000;
-
-  private static final Logger logger = LoggerFactory.getLogger(ModTenantAPI.class);
+  private static final String LOAD_SAMPLE_PARAMETER = "loadSample";
```

Comment:
As this is default data, let's call it that, and not sample

Reply:
sure, i will rename it